.github/workflows/scratch-org-sfdx-ci-pr.yml

```diff
@@ -80,10 +86,18 @@ jobs:
                   tar xJf sfdx-linux-amd64.tar.xz -C sfdx-cli --strip-components 1
                   ./sfdx-cli/install
 
+            # Install Salesforce CLI Scanner
+            - name: Install Salesforce CLI Scanner
+              run: 'echo y | sfdx plugins:install @salesforce/sfdx-scanner'
```

Comment:
We will need the `echo y` because the plugin is not signed yet. There is a work item for it in GUS and an issue here (https://github.com/forcedotcom/sfdx-scanner/issues/241).

We will take `echo y` off once we have a signed plugin